x-pack/plugins/security_solution/cypress/screens/timeline.ts

```diff
@@ -73,3 +73,11 @@ export const TIMESTAMP_HEADER_FIELD = '[data-test-subj="header-text-@timestamp"]
 export const TIMESTAMP_TOGGLE_FIELD = '[data-test-subj="toggle-field-@timestamp"]';
 
 export const TOGGLE_TIMELINE_EXPAND_EVENT = '[data-test-subj="expand-event"]';
+
+export const TIMELINES_TABLE = '[data-test-subj="timelines-table"]';
+
+export const BULK_ACTIONS = '.siemUtilityBar__action button';
```

Comment:
It is possible to add a `data-test-subj` locator so we can use it instead of the class?

Reply:
Yup! Thanks a lot, I'll change those selectors.